content.md

```diff
@@ -1001,6 +982,34 @@ the blood bank.
 ![](bin/media/image33.png)
 
 
+### **Action Card (or Card in Play)**
+
+- **Who can perform an action by default:** Any ready minion. A minion
+cannot perform an action with the same action card from hand or in play
+more than once each turn, even if they unlock.
+- **Cost:** As listed on the card.
+- **Default target:** As listed on the card.
+- **Default stealth:** Unless otherwise noted on the card, these actions
+are at 0 stealth.
+- **Effect:** If the action is successful, it resolves as indicated on
+the card. If the card describes a special version of a basic action,
+then all the rules that apply to the basic action apply, except as
+otherwise noted on the card.
+
+
+
+![](bin/media/image41.png)
+
+
+
+> **ADVANCED RULES**\
+> \
+> **Become Anarch**
+> - Who can become Anarch: Any ready untitled non-Anarch vampire.
```

Comment:
Prefer `non-titled` to `untitled`.  I know this passage moved, but cards use `non-titled`, we should use the same language in the rulebook.
```suggestion
> - Who can become Anarch: Any ready non-titled non-Anarch vampire.
```